tbx/project_styleguide/templates/patterns/organisms/footer/footer.html

```diff
@@ -1,53 +1,58 @@
 {% load navigation_tags wagtailcore_tags wagtailimages_tags %}
 
-<footer>
-    {% with global_settings=settings.navigation.NavigationSettings %}
-        {% if global_settings.footer_logos %}
-            <ul>
-                {% for item in global_settings.footer_logos %}
-                    {% with logo=item.value %}
-                        <li>
-                            {% with link=logo.link.0.value %}
-                                {% if link %}
-                                    <a href="{{ link.url }}" title="{{ link.link_text }}">
-                                        {% image logo.image height-90 format-webp loading="lazy" alt=link.link_text %}
-                                    </a>
-                                {% else %}
-                                    {% image logo.image height-90 format-webp loading="lazy" %}
-                                {% endif %}
-                            {% endwith %}
-                        </li>
-                    {% endwith %}
-                {% endfor %}
-            </ul>
-        {% endif %}
-    {% endwith %}
+<footer class="footer">
+
+    <div class="grid footer__top ">
+        <span class="grid__footer-line footer__line"></span>
+        {% include "patterns/molecules/footer-cta/footer-cta.html"%}
 
-    {% footerlinks %}
+        <div class="grid__footer-links">
+            {% footerlinks %}
+            <p class="footer__copyright">© Torchbox {% now "Y" %}</p>
 
-    <ul>
-        <li>
-            <a aria-label="Follow us on Twitter" href="https://twitter.com/torchbox">
-                <svg width="24" height="24" aria-hidden="true">
-                    <use xlink:href="#twitter" />
-                </svg>
-            </a>
-        </li>
-        <li>
-            <a aria-label="Connect with us on LinkedIn" href="https://www.linkedin.com/company/torchbox">
-                <svg width="24" height="24" aria-hidden="true">
-                    <use xlink:href="#linkedin" />
-                </svg>
-            </a>
-        </li>
-        <li>
-            <a aria-label="Follow us on Instagram" href="https://www.instagram.com/torchboxltd/">
-                <svg width="24" height="24" aria-hidden="true">
-                    <use xlink:href="#instagram" />
-                </svg>
-            </a>
-        </li>
-    </ul>
+            <ul class="footer__socials">
+                <li>
+                    <a aria-label="Follow us on Twitter" href="https://twitter.com/torchbox" class="footer__social-link">
```

Comment:
These social anchors are missing our focus styles - they've got the browser default at the mo : )

Reply:
There's a mixin for the focus styles here: https://github.com/torchbox/torchbox.com/blob/integration/sprint-3/tbx/static_src/sass/config/_mixins.scss#L122

Reply:
fixed in bd61252d5f5344393bd91350cb5ca611d96828f7